superset/connectors/base/models.py

```diff
@@ -404,7 +404,7 @@ def handle_single_value(value: Optional[FilterValue]) -> Optional[FilterValue]:
                     return utils.cast_to_num(value)
                 if value == NULL_STRING:
                     return None
-                if value == "<empty string>":
+                if value == EMPTY_STRING:
```

Comment:
Can we remove these now, both `EMPTY_STRING` and `NULL_STRING`? Or are these needed due in some legacy component?

Reply:
I think it's used to adapt some legacy Select component.